### ISSUE TITLE
Implement version + release schedule 

### ISSUE THREAD

**Issue by JohnGriffiths** [NeuroTechX/EEG-ExPy#216]
(Note: general eegnb dev discussions to be relocated to github DIscussions on this page. 

We are going to merge the develop branch to master, which contains two main new components: the refactored experiment class and the analysis report function. 

As part of doing this, we will implement a more structured release schedule process, which will allow us to more easily keep on top of changes and comfortably execute releases / merge updates to master that *might* possibly cause backwards compatibility issues. 

Overall tasks:

- [x] Confirm we are ready to move develop branch contents into master   (Taha)
   - [ ] Check the code runs (again) (Taha)
   - [ ] Check for any API changes from previous version, and either adjust if possible or document (Taha)
   - [ ] Write docs on new features (analysis report; maybe others) (Taha)
   - [ ] Update docs on anything major that's missing (Taha)
   - [ ] Check docs in general; esp. installation (Taha; John)
- [ ] Write doc describing this version plan  ( John )
- [x] Create new branch from current master, called 0.2 (current version)  (John) 
- [ ] Create new branch from develop, called 0.3 (new version)  (John)
   - [ ] Check requirements are up to date
   - [ ] Make changelog file
   - [ ] Remove Parv submodule [Ore]
- [ ] Merge 0.3 to master  (John0)
- [ ] Add pypi installation for 0.3 onwards   (Ore)
- [ ] Add multi-version doc build configuration (c.f. github.com/mne-tools.github.io) (Taha)

**Comment by tmorshed** [comment]
**Problem:**
Regarding the first action item, we see that `Develop` fails building the docs. 

**Investigation:**
Started a new codespaces instance and checked it all. Still failed. The only thing that needed to be changed was the linux version. Seems like GH moved to Ubuntu 22.04 for actions since Nov. 2022 (which is around the time that this issue started). 

**Solution**:
Updating the version number of the `wxpython` wheel to 22.04. i.e.:
```
pip install -U -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-18.04 wxPython
```
must become 
```
pip install -U -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-22.04 wxPython
```
Will try the update and see if it fixes the issue.

**Update:** No need to do this
Turned out I was using `Develop`. the actual `develop` builds the docs okay. [Ben's commit](https://github.com/NeuroTechX/eeg-notebooks/commit/30fb77c26b3121532487ca3c53537b2f33aff6b3) (on #210) has already fixed that.

**Comment by Parvfect** [comment]
*Problem with respect to Docs*

Docs tutorials for Running Experiments deal with importing individual experiements and then running them as per, 
![image](https://user-images.githubusercontent.com/65726543/213483431-37ad8030-9e99-46f7-ada6-e8203152683b.png)

However, with the Experiment Class Refactor #184, with classes and objects and such, the call changes slightly and will not work with the current code. Going to address this in a new pull request to master once the merge is done to fix the problem.
